docs/conf.py

```diff
@@ -25,9 +25,9 @@
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
 #
-# import os
-# import sys
-# sys.path.insert(0, os.path.abspath('.'))
+import os
```

Comment:
This imports `os` twice as it's done at L36 already.

Reply:
Fixed in [3c19b14](https://github.com/Qiskit/qiskit-ibmq-provider/pull/1166/commits/3c19b142d1d483dc95550ee079b7059b560d6167)